packages/cashier/src/Components/error-component.jsx

```diff
@@ -46,8 +46,8 @@ const ErrorComponent = ({
                           refresh_message,
                       ]
             }
-            redirect_url={redirect_to}
-            redirect_label={redirect_label || <Localize i18n_default_text='Refresh' />}
+            redirect_urls={[redirect_to]}
+            redirect_labels={[redirect_label || <Localize key={0} i18n_default_text='Refresh' />]}
```

Comment:
Why do we need key={0}?

Reply:
It is throwing error without the key for the element.